Kernel/Graphics/Intel/DisplayConnectorGroup.h

```diff
@@ -25,9 +25,6 @@ class IntelDisplayConnectorGroup : public RefCounted<IntelDisplayConnectorGroup>
     friend class IntelNativeGraphicsAdapter;
 
 public:
-    enum class Generation {
```

Comment:
another squash comment, can we have Generation in the defintions file at the same time we introduce the concept of Generations? ` Kernel/IntelGraphics: Move Generation enum to the Definitions file `